src/KafkaInstanceDrawer/DetailsTab.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import {
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+} from "@patternfly/react-core";
+import { DetailsTabAlert } from "./DetailsTabAlert";
+import { format } from "date-fns";
+
+type DetailsTabProps = {
+  id: string;
+  created_at: Date;
+  updated_at: Date;
```

Comment:
```suggestion
  createdAt: Date;
  updatedAt: Date;
```

lowerCamelCase please